courses/models.py

```diff
@@ -153,6 +167,22 @@ class Meta:
     multiple_languages = property(multi_language)   #multiple_languages is a property on KnowledgeBase
     multi_language.admin_order_field = '-modified_at'  #multi_language is ordered by reverse modified_at at the admin panel
 
+    """
+    Why not this convention?
```

Comment:
for using short description in admin site, this is from the Django documentation
"Elements of list_display can also be properties. Please note however, that due to the way properties work in Python, setting short_description on a property is only possible when using the property() function and not with the @property decorator."